libclamav/matcher-ac.c

```diff
@@ -2915,7 +2915,7 @@ cl_error_t cli_ac_addsig(struct cli_matcher *root, const char *virname, const ch
         return ret;
     }
 
-    if (new->offdata[0] != CLI_OFF_ANY && new->offdata[0] != CLI_OFF_ABSOLUTE && new->offdata[0] != CLI_OFF_MACRO) {
+    if (new->offdata[0] != CLI_OFF_ANY &&new->offdata[0] != CLI_OFF_ABSOLUTE &&new->offdata[0] != CLI_OFF_MACRO) {
```

Comment:
Is this spacing a result of clang-format?  I think this makes it harder to read.

Reply:
Huh, I hadn't noticed that.  I agree it makes it harder to read, and I have no idea why clang-format would remove those. Will investigate.

Reply:
It was definitely clang-format weirdness.   I haven't seen that one before.  I added parenthesis and new lines and now i think it's even easier to read and clang-format doesn't mess with it.